sonar-text-plugin/src/test/java/org/sonar/plugins/secrets/utils/UpdatingSpecificationFilesGenerator.java

```diff
@@ -59,11 +66,17 @@ class UpdatingSpecificationFilesGenerator {
   private final Charset charset = StandardCharsets.UTF_8;
   private static final Logger LOG = LoggerFactory.getLogger(UpdatingSpecificationFilesGenerator.class);
 
+  @RegisterExtension
+  LogTesterJUnit5 logTester = new LogTesterJUnit5();
+
   // Suppress warning, as there are no assertions inside here
   @Test
   @SuppressWarnings("java:S2699")
   void firstStep() {
     writeSpecificationFileDefinition();
+    assertThatNoException().isThrownBy(SpecificationLoader::new);
```

Comment:
Won't this always succeed as we capture deserialization exceptions inside `SpecificationLoader`?

Reply:
Yes this assert can be removed